assets/queries/dockerfile/secrets_env/query.rego

```diff
@@ -0,0 +1,52 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command[j]
+  resource.Cmd == "env" 
+  checkSecret(resource) == true
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("ENV=%s", [resource.Value[0]]),
+                "issueType":		"IncorrectValue",  #"MissingAttribute" / "RedundantAttribute"
+                "keyExpectedValue": "Dockerfile doesn't contain secrets",
+                "keyActualValue": 	"Dockerfile contains secrets"
```

Comment:
Try to specify the command to make it easier for the user to track the faulty command